tests/core/htmldataprocessor/customtag.js

```diff
@@ -0,0 +1,68 @@
+/* bender-tags: editor */
+
+( function() {
+	var elements = [
+		'object',
+		'embed',
+		'param',
+		'html',
+		'head',
+		'body',
+		'title'
+	],
+	extraAllowedContent = '';
+
+	CKEDITOR.tools.array.forEach( elements, function( item ) {
```

Comment:
This could be easily replaced with a one liner, like:

`elements.join( 'foo[*]{*}(*);' ) + 'foo[*]{*}(*);' or `elements.concat( [ '' ] ).join( 'foo[*]{*}(*);' )`  (though I like second more because it has less repetition, conat `[ '' ]` thing might not be obvious in a first place.

From there `extraAllowedContent` var can be inlined directly into config object.